qc/check_polygon_layers.py

```diff
@@ -1,5 +1,7 @@
 import os
+import io
 import json
+import pandas as pd
```

Comment:
imports should be sorted. 
1. standard packages
2. 3rd party
3. local imports